config.ts

```diff
@@ -1,44 +1,38 @@
-import { generateDeploymentConfig } from "scripts";
-const config = generateDeploymentConfig("plh_facilitator_mx");
+import { extendDeploymentConfig } from "scripts";
+const config = extendDeploymentConfig({ name: "plh_facilitator_mx", parent: "plh_facilitator" });
 
-/**
- * The default config should ideally be a superset of any extended configs
- * to allow for easier post-processing
- */
+config.google_drive.sheets_folder_ids =  ["1k8zJfBeOm18gYDsk0XmdUm7OFtZDUY9R", "1rFg8qsmb3xSOXw_9Iwxk8RgSmyjqiao9"];
+config.google_drive.assets_folder_ids = ["1KcHDI7O4o2_FZ_YlXsz-8OqN3ehsfdVf", "1tr76W9OusHHkjJ4DNR84x-B26VgGCSFy"];
 
-config.google_drive = {
-  sheets_folder_ids: ["1k8zJfBeOm18gYDsk0XmdUm7OFtZDUY9R", "1rFg8qsmb3xSOXw_9Iwxk8RgSmyjqiao9"],
-  assets_folder_ids: ["1KcHDI7O4o2_FZ_YlXsz-8OqN3ehsfdVf", "1tr76W9OusHHkjJ4DNR84x-B26VgGCSFy"],
+config.git = {
+  content_repo: "https://github.com/IDEMSInternational/plh-facilitator-app-mx-content.git",
+  content_tag_latest: "1.1.35",
 };
 
 config.android = {
+  app_id:'international.idems.plh_facilitator_mx',
+  app_name:'Formando Conciencia+',
   splash_asset_path: "./app_data/assets/android/splash.png",
   icon_asset_path: "./app_data/assets/android/icon.png",
   icon_asset_foreground_path: "./app_data/assets/android/icon-foreground.png",
   icon_asset_background_path: "./app_data/assets/android/icon-background.png",
 };
 
 config.web.favicon_asset = "images/logos/bird_on_light.svg";
-
-config.git = {
-  content_repo: "https://github.com/IDEMSInternational/plh-facilitator-app-mx-content.git",
-  content_tag_latest: "1.1.34",
-};
-
 config.api.db_name = "plh_facilitator_mx"
 
+// Hacky fix to point extended deployment to translations within its own repo
+config.translations.translated_strings_path = "./app_data/translations_source/translated_strings";
+
+// Hacky fix to point extended deployment to content within its own repo
 config.app_data.output_path = "./app_data";
```

Comment:
@jfmcquade Without repeating this line within the extended deployment config, I was only seeing the parent deployment content, not the child's. Does that make sense to you?